src/main/java/org/spongepowered/common/bridge/server/MinecraftServerBridge.java

```diff
@@ -46,4 +47,7 @@
     boolean bridge$performAutosaveChecks();
 
     ServerTimingsHandler bridge$timingsHandler();
+
+    SpongeUserManager bridge$userManager();
```

Comment:
Wonder if this should be put onto `SpongeServer`? I have a, albeit, weird separation going on there.

Reply:
Actually, yes, that would be a perfect candidate to do so.